examples/helloworld/lib/helloworld_app.ex

```diff
@@ -7,6 +7,7 @@ defmodule HelloworldApp do
       GrpcReflection
     ]
 
+    Protobuf.load_extensions()
```

Comment:
We are already asking the users to add our supervision subtree to their application by placing `GrpcReflection` to their supervision children.  We can move this into our own modules supervision tree as an initialization step and not ask end-users to modify their application for our need

Reply:
You are right, I moved it to `Agent.start_link/2`